python/mxnet/ndarray/sparse_ndarray.py

```diff
@@ -217,15 +214,17 @@ def _aux_data(self, i):
 
 # pylint: disable=abstract-method
 class CSRNDArray(BaseSparseNDArray):
-    """A CSRNDArray represents a NDArray as three separate arrays: `data`,
+    """A sparse representation of tensor in standard CSR format.
```

Comment:
Do we need to change `tensor` to `matrix` since csr only represents matrices?

Also, "in **the** standard CSR format."